arch/posix/include/asm_inline_gcc.h

```diff
@@ -0,0 +1 @@
+/* EMTPTY ON PURPOSE. Why do the intel and ARM arch have 2 versions of it? */
```

Comment:
@nashif  has zephyr a convention on this.

Reply:
I was a bit puzzled when I saw those other archs having 2 versions of the file which share a lot of the content.. I would be happy if somebody could give an explanation.
In the meanwhile I created the empty one so there would no problems with includes.